### ISSUE TITLE
How to pass parameters?

### ISSUE THREAD

**Issue by EthraZa** [pestphp/pest#252]
I'm using Laravel Zero.

The command:
```
# ./fetchattach show app.name
'Fetchattach'
```
The line dumping this result is:
```
$this->info(var_export(config($arg??'', $default)));
```
I'm trying to write the most simple pest to it, but can't get it to work and didn't find an example passing some parameters in the docs:
```
test('show command', function () {
    $this->artisan('show', ['app.name'])
         ->expectsOutput('\'Fetchattach\'')
         ->assertExitCode(0);
});
```
I've tryed to pass the 'app.name' parameter in a bunch of different ways but simple can't find how to get it done. 
So, how to pass parameters to methods being tested?

---
Bonus question: There is a way to dump how our test are calling the methods being tested and what is the actual output?

**Comment by octoper** [comment]
cc @owenvoke

**Comment by octoper** [comment]
I think that's not related to Pest it's related to the Laravel framework

**Comment by owenvoke** [comment]
The 2nd Param in `artisan()` needs to be a keyed array. So if your signature was `show {key}`, you would do:
```php
$this->artisan('show', ['name' => 'app.name']);
```

**Comment by EthraZa** [comment]
Now I noticed the string `'Fetchattach'` being dumped as the test 1st output of the test and then I find out it is being displayed there not as a feature of Pest, but because I was using var_export() without "true" as second param, so it was dumping the string to the Std Output instead of returning it to the info() method.

So, today I learned two things:
1- how to pass params to artisan inside Pest (it would be cool to have at least 1 example of this in the Pest docs) 
2- how to debug the output being tested inside Pest (dumping it direct to std output from the code).

I got it "passed" now, thank you.
